### PR TITLE
Feature leaderboard highlight

### PR DESCRIPTION
**Pull request by AthishThayalan**
Top scorer is highlighted in green and current user is highlighted in grey.

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -103,7 +103,7 @@ const App = () => {
 
         <Route
           path="/leaderboard"
-          element={<Leaderboard users={fetchedTribe} />}
+          element={<Leaderboard users={fetchedTribe} currentUserID={userUID} />}
```
Comment (Charlie-robin) [inline src/App.tsx:106]: You can remove one of the Leaderboard Routes as you have two.

File: src/components/LeaderboardCard/LeaderboardCard.scss [hunk 2]
```diff
@@ -33,10 +35,19 @@
     color: $primary-color;
   }
 }
+.leaderboard-card--current-user {
+  background-color: $light-grey-color;
+}
+.leaderboard-card--first {
+  background-color: $primary-color;
+  p {
```
Comment (Charlie-robin) [inline src/components/LeaderboardCard/LeaderboardCard.scss:43]: Can you add a class name here please?

File: src/components/LeaderboardCard/LeaderboardCard.test.tsx [hunk 3]
```diff
@@ -18,28 +21,34 @@ describe("should pass the LeaderboardCard tests", () => {
...
+        profileImage={"Test Image"}
+        totalScore={100}
+        isFirstCard={true}
+        currentUserID=""
+        userID=""
+      />
+    );
+    const capitalName = screen.getByText("Test Name");
+    expect(capitalName).toBeTruthy();
+  });
```
Comment (Charlie-robin) [inline src/components/LeaderboardCard/LeaderboardCard.test.tsx:53]: You will want to add in some new tests for your functionality

File: src/pages/Leaderboard/Leaderboard.tsx [hunk 4]
```diff
@@ -4,31 +4,43 @@ import { UserProfile } from "../../mockData/mockTribe";
...
+        if (!a.name) return 1;
+        if (!b.name) return -1;
         return a.name.localeCompare(b.name);
       }
       return 0;
     });
     return sortScoreAndName;
   };
 
+  console.log("Users: " + users);
```
Comment (Charlie-robin) [inline src/pages/Leaderboard/Leaderboard.tsx:27]: You can remove the log

File: src/components/LeaderboardCard/LeaderboardCard.tsx [hunk 5]
```diff
@@ -1,27 +1,46 @@
...
+const LeaderboardCard = ({
+  name,
+  profileImage,
+  totalScore,
+  isFirstCard,
+  currentUserID,
+  userID,
+}: LeaderboardCardProps) => {
+  const myName = capitalisedFirstLetters(name);
+  console.log(`Currentscore: "${name}\nCurrent User ID: ${userID} `);
```
Comment (Charlie-robin) [inline src/components/LeaderboardCard/LeaderboardCard.tsx:22]: You can remove the log

**Review by Charlie-robin — CHANGES_REQUESTED** [review]
Some comments to work on and also you will need to update the stories file for the LeaderboardCard to include the new props you are passing in

File: src/components/LeaderboardCard/LeaderboardCard.scss (hunk 2 above)
Comment (Divya-Manivannan1) [reply src/components/LeaderboardCard/LeaderboardCard.scss]: checked

File: src/components/LeaderboardCard/LeaderboardCard.test.tsx (hunk 3 above)
Comment (Divya-Manivannan1) [reply src/components/LeaderboardCard/LeaderboardCard.test.tsx:53]: verified

**Review by Charlie-robin — APPROVED** [review]

**Pull request merged**